examples/inspector/src/viewer/new-app.tsx

```diff
@@ -80,23 +81,47 @@ export default function CoJsonViewerApp() {
         websocket: new WebSocket("wss://cloud.jazz.tools"),
         role: "server",
       });
-      const node = await LocalNode.withLoadedAccount({
-        accountID: currentAccount.id,
-        accountSecret: currentAccount.secret,
-        sessionID: crypto.newRandomSessionID(currentAccount.id),
-        peersToLoadFrom: [wsPeer],
-        crypto,
-        migration: async () => {
-          console.log("Not running any migration in inspector");
-        },
-      });
+      let node;
+      try {
+        node = await LocalNode.withLoadedAccount({
+          accountID: currentAccount.id,
+          accountSecret: currentAccount.secret,
+          sessionID: crypto.newRandomSessionID(currentAccount.id),
+          peersToLoadFrom: [wsPeer],
+          crypto,
+          migration: async () => {
+            console.log("Not running any migration in inspector");
+          },
+        });
+      } catch (err: any) {
+        const invalidAccountIndex = accounts.findIndex(
+          (acc) => acc.id === currentAccount.id,
+        );
+        accounts.splice(invalidAccountIndex, 1);
+
+        setAccounts(accounts);
```

Comment:
The inspector doesn't have local persistence so an account might fail to load because of connectivity issues.

We can't be sure that's a broken account, and since this effects always run on the inital load we might end up removing from the list some valid accounts.

We should find a way to delete only the "never loaded accounts"